app/pages/project/networking/VpcPage/tabs/VpcFirewallRulesTab.tsx

```diff
@@ -74,6 +81,18 @@ export const VpcFirewallRulesTab = () => {
   // the whole thing can't be static because the action depends on setEditing
   const columns = useMemo(() => {
     return [
+      colHelper.accessor('name', {
+        header: 'Name',
+        cell: (info) => (
+          <>
+            <button
+              className="peer absolute inset-0"
+              onClick={() => setEditing(info.row.original)}
+            />
+            <div className="peer-hover:underline">{info.getValue()}</div>
```

Comment:
Bad news — this trick isn't working for me in Firefox. The button has zero width.

<img width="1095" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/fd81828e-bf1c-4911-9a28-2f8b5a515667">

Reply:
Can probably replace it with the same trick we're using on the clickability PR

Reply:
I can create a `buttonCell` component, but really I think the best solution here would to change it to route tabs, reuse `linkCell`, and allow the edit modal to be linkable.

Reply:
Will redo this in a followup.